builtin/providers/google/data_source_google_compute_subnetwork.go

```diff
@@ -0,0 +1,86 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/googleapi"
+)
+
+func dataSourceGoogleComputeSubnetwork() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleComputeSubnetworkRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"self_link": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ip_cidr_range": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"network_self_link": &schema.Schema{
```

Comment:
Hmm, so there's an equivalent field in the subnetwork resource which is just called `network`. I like that your name is more descriptive, but for consistency's sake it may be better to just go with that.